pkg/config/config.go

```diff
@@ -39,4 +39,18 @@ type Config struct {
 	HTTPTransport *http.RoundTripper
 	UserAgent     string
 	Region        RegionType
+
+	// LogLevel can be one of the following values:
+	// "panic", "fatal", "error", "warn", "info", "debug", "trace"
+	LogLevel string
+	LogJSON  bool
+	Logger   Logger
+}
+
+// Logger interface implements a simple logger.
+type Logger interface {
+	Error(string, ...interface{})
+	Info(string, ...interface{})
+	Debug(string, ...interface{})
+	Warn(string, ...interface{})
```

Comment:
We should move this into the `logging` package and reference it here instead.

Reply:
Logging is internal, I had moved it here so it was more discoverable for folks implementing it.